pom.xml

```diff
@@ -16,31 +16,26 @@
  ~  specific language governing permissions and limitations
  ~  under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.wso2.carbon.connector</groupId>
-    <artifactId>org.wso2.carbon.connector.sharepoint</artifactId>
+
+<!-- Generated on 31-Thu, 10, 2024 21:21:45+0530 -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <groupId>org.wso2.carbon.esb.connector</groupId>
+    <artifactId>sharepoint</artifactId>
+    <version>1.1.0</version>
+
+    <name>WSO2 Carbon - Sharepoint Connector</name>
+    <url>https://wso2.com</url>
     <packaging>jar</packaging>
-    <version>1.0.5</version>
-    <name>WSO2 Carbon - Mediation Library Connector For sharepoint</name>
-    <url>http://wso2.org</url>
+    <modelVersion>4.0.0</modelVersion>
+
     <properties>
         <connector.name>sharepoint</connector.name>
-        <product.ei.version>6.4.0</product.ei.version>
-        <carbon.kernel.version>4.4.17</carbon.kernel.version>
-        <automation.framework.utils.version>4.4.2</automation.framework.utils.version>
-        <automation.framework.version>4.4.3</automation.framework.version>
-        <emma.version>2.1.5320</emma.version>
-        <synapse.version>2.1.7-wso2v19</synapse.version>
-        <carbon.mediation.version>4.6.19</carbon.mediation.version>
-        <json.version>2.0.0.wso2v1</json.version>
-        <org.testng.version>6.1.1</org.testng.version>
-        <jets3t.version>0.9.4</jets3t.version>
-        <bouncycastle.version>1.55</bouncycastle.version>
-        <rampart.apache.version>1.6.1-wso2v18</rampart.apache.version>
-        <rampart.wso2.version>1.6.1.wso2v18</rampart.wso2.version>
-        <integration.base.version>1.0.3</integration.base.version>
-        <skip-tests>true</skip-tests>
```

Comment:
The tests are skipped by default. Shall we maintain the same behaviour in the new model as well? In that way, we don't have to explicitly skip the tests when building the connector.

Reply:
`mvn clean install -Dmaven.test.skip=true` is used to skip compiling tests as well, because otherwise it will fail, will this does not seem to have the same behavior.
<img width="1488" alt="Screenshot 2024-11-07 at 20 44 52" src="https://github.com/user-attachments/assets/b755e67a-80f9-4a41-918c-79b6b592853b">